webdriverio/webdriver/examples/visual-e2e/README.md

```diff
@@ -17,7 +16,7 @@ You can run your tests on Sauce Labs US DC with this command
     
 Run cross-platform, across multiple resolutions using:
 
-`npm run test.visual.crossplatform`
+    npm run test.visual.crossplatform
```

Comment:
@wswebcreation that's in the Screener documentation

Reply:
That's true, but shouldn't our examples also explain this, because otherwise people would already use the correct commands and so on if they have read the Screener docs